YUIScriptProcessor syntax error

### Expected Behaviour

No errors in the author error.log

### Actual Behaviour

`[ERROR] null
02.12.2019 10:19:02.036 *ERROR* [0:0:0:0:0:0:0:1 [1575278342019] GET /libs/commerce/gui/components/common/cifcategoryfield/clientlibs.min.js HTTP/1.1] com.adobe.granite.ui.clientlibs.impl.HtmlLibraryManagerImpl Error during assembly of library.
org.mozilla.javascript.EvaluatorException: Compilation produced 1 syntax errors.
	at com.adobe.granite.ui.clientlibs.impl.YUIScriptProcessor$1.runtimeError(YUIScriptProcessor.java:146) [com.adobe.granite.ui.clientlibs:1.0.76.CQ640-B0003]
	at org.mozilla.javascript.Parser.parse(Parser.java:396)
	at org.mozilla.javascript.Parser.parse(Parser.java:340)`

#### Steps to Reproduce

Enable minification under  'Adobe Granite HTML Library Manager'

<img width="455" alt="Screenshot 2019-12-02 at 10 20 45" src="https://user-images.githubusercontent.com/4804496/69947213-7b7fd000-14ed-11ea-9d33-a71e618f7850.png">

#### Platform and Version

Adobe Experience Manager (6.4.5.0) & connector 0.7.0

--- a/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/authoring/editor/pagepreview/clientlibs/pagepreview.js
+++ b/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/authoring/editor/pagepreview/clientlibs/pagepreview.js
@@ -77,5 +77,5 @@ window.CIF.PagePreview = {};
     Granite.$(document).on('cifProductPickerSelection', relPdpPreview, handlePreview);
     Granite.$(document).on('cifCategoryPickerSelection', relPlpPreview, handlePreview);
 
-    window.CIF.PagePreview = { handlePreview, createPreviewUrl };
+    window.CIF.PagePreview = { handlePreview: handlePreview, createPreviewUrl: createPreviewUrl };
 })(window, document, Granite);

--- a/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/clientlibs/cifcategorypicker.js
+++ b/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/clientlibs/cifcategorypicker.js
@@ -95,5 +95,5 @@ window.CIF.CifCategoryPicker = {};
 
     Granite.$(document).on('click', relActivator, clickActivator);
 
-    window.CIF.CifCategoryPicker = { clickActivator, relActivator };
+    window.CIF.CifCategoryPicker = { clickActivator: clickActivator, relActivator: relActivator };
 })(window, document, Granite);

--- a/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifpicker/clientlibs/cifpicker.js
+++ b/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifpicker/clientlibs/cifpicker.js
@@ -103,5 +103,5 @@ window.CIF.CifPicker = {};
         }
     };
 
-    window.CIF.CifPicker = { cifPicker, getState, show, close };
+    window.CIF.CifPicker = { cifPicker: cifPicker, getState: getState, show: show, close: close };
 })(window, document, Granite);

--- a/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifproductfield/clientlibs/cifproductpicker.js
+++ b/content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifproductfield/clientlibs/cifproductpicker.js
@@ -95,5 +95,5 @@ window.CIF.CifProductPicker = {};
 
     Granite.$(document).on('click', relActivator, clickActivator);
 
-    window.CIF.CifProductPicker = { clickActivator, relActivator };
+    window.CIF.CifProductPicker = { clickActivator: clickActivator, relActivator: relActivator };
 })(window, document, Granite);

